### PR TITLE
actix web client instead of hyper, one client instance per thread

### PR DESCRIPTION
**Pull request by estin**
Hello!
This PR allow to share actix web client per request for one thread.
For each thread initialized one client and reused for each request served by this thread.

As you I don't find how to share one client for all threads.
But if each thread will have one client it is great too )

And I can't check e2e tests
```
$ docker-compose up -d
Pulling dali (docker.pkg.github.com/olxgroup-oss/dali:latest)...
ERROR: Get "https://docker.pkg.github.com/v2/olxgroup-oss/dali/manifests/latest": no basic auth credentials
```

Sorry for my poor English.

### CODE REVIEW

**Comment by asonix** [comment]
you don't need to add `awc` as a dependency, you can use `actix_web::client::{Client, Connector}`

**Comment by estin** [comment]
Thanks! Fixed

File: src/commons/http.rs [hunk 1]
```diff
@@ -2,45 +2,22 @@
...
+        .map_err(move |e| {
+            let error_str = format!("{}", e).replace("\"", "\\\"");
+            error!("Error to send request: {}", error_str);
+            actix_web::error::InternalError::new(String::from("Error to send request"), actix_http::http::StatusCode::INTERNAL_SERVER_ERROR)
+        })?;
     let status = response.status();
-    if response.status().is_success() {
-        to_bytes(response.into_body()).await.map_err(move |e| {
+    if status.is_success() {
+        response.body().await.map_err(move |e| {
```
Comment (augustocdias) [inline src/commons/http.rs]: We need to set the `limit` from the body as it is quite low. We could add a configuration for that 
https://docs.rs/awc/2.0.0-alpha.1/awc/struct.MessageBody.html#method.limit

**Comment by augustocdias** [comment]
Can you add a [feature flag](https://doc.rust-lang.org/1.29.0/book/first-edition/conditional-compilation.html) for it and defaulting to hyper. Otherwise I'll need quite some time to test it as we had issues with `awc` ddosing our DNS server.

**Comment by estin** [comment]
I will do it - feature flag and  settings for awc setup
It will take some time

**Comment by estin** [comment]
Update:
 - feature flags `hyper_client`  and `awc`, raises `compile_error` if both features are enabled
 - new configuration  `http_client_max_size_of_payload`

Introduce one API for http client module.

```
$ cargo check --no-default-features --features awc
$ cargo run --no-default-features --features awc
```

**Pull request merged**